components/popover_list_members/popover_list_members.jsx

```diff
@@ -220,7 +220,6 @@ export default class PopoverListMembers extends React.Component {
         return (
             <div id='channelMember'>
                 <OverlayTrigger
-                    trigger={'hover'}
```

Comment:
You've been using `trigger={['hover', 'click']}` on all other `OverlayTrigger`'s so far, just wanted to check that removing this to use defaults wasn't an oversight ... I'm not sure the context, so I could have missed something.